### ISSUE TITLE
Possibility to add custom css for numeric strings

### ISSUE THREAD

**Issue by qikkeronline** [nathanford/type.js#1]
Cool project! Would love a way to be able to change the properties of numeric strings, as we often have webfonts where the numbers are aligned differently then the rest of the glyphs.

**Comment by rogeriomarques** [comment]
+1

**Comment by nathanford** [comment]
That’s a very interesting idea. What kind of control would you want? Do you have an example of the method to fix it?

**Comment by qikkeronline** [comment]
I guess it would be most useful to wrap the characters with an element with classname attached to it, so we can tweak it via regular CSS, as every webfont is different and this way you could differentiate a bit easier on sites with multiple fonts. I don't have a method to fix it at the moment, and also no time for it :) if you still need thoughts on it later I'll be checking in here regularly!

**Comment by nathanford** [comment]
Yeah, sorry: I was wondering what kind of tweaks you need to make. Like: adjust the letter spacing of numerals, or adjusting the vertical alignment?

**Comment by rogeriomarques** [comment]
Vertical alignment would be fantastic. Also, some way to control numerical separators (thousands, decimals).

**Comment by qikkeronline** [comment]
It's mainly vertical alignment; but I think in some cases letter-spacing control would also be useful (in case that the numeric string is longer than one digit) :)
